OpenImis.ModulesV2/InsureeModule/Repositories/InsureeRepository.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using OpenImis.DB.SqlServer;
+using OpenImis.ModulesV2.InsureeModule.Helpers;
+using OpenImis.ModulesV2.InsureeModule.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Data.SqlClient;
+using System.Diagnostics;
+using System.Text;
+
+namespace OpenImis.ModulesV2.InsureeModule.Repositories
+{
+    public class InsureeRepository : IInsureeRepository
+    {
+        private IConfiguration _configuration;
+
+        public InsureeRepository(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public GetInsureeModel Get(string chfid)
```

Comment:
This is for GetEnquire as it calls the uspPolicyInquiry SP. We need another call for Get(chfid) that will only do a simple select and get the information in GetInsureeModel

Reply:
Done